admin_ui/src/views/RowListing.vue

```diff
@@ -114,15 +114,15 @@
                                             <span
                                                 class="link"
                                                 v-else-if="
-                                                    isForeignKey(name) & (row[name] !== null)
+                                                    isForeignKey(name) & (row[pkName] !== null)
                                                 "
                                             >
                                                 <router-link
                                                     :to="{
                                                         name: 'editRow',
                                                         params: {
                                                             tableName: getTableName(name),
-                                                            rowID: row[name]
+                                                            rowID: row[pkName]
```

Comment:
I also thought so before, but the `rowID: row[pkName]` is the primary key of the `Review` table, not `Review.movie.id`.
I wrote in the comment the changes I think we need to make in Piccolo API to get correct results.

Reply:
You're right - it took a while for me to realise this. My solution doesn't work.